File: student.py
class student:
    studentID = 0
    fName = ""
    lName = ""
    status = ""

    termStart = ""
    termEnd = ""
    peopleSoftID = ""

    dob = ""
    phone = 0
    email = ""
    address = ""

    def __init__(self,id,fName,lName):
        self.studentID = id
        self.lName = lName
        self.fName = fName

    def __init__(self,id,fName,lName,status,termStart,termEnd,peopleSoft,dob,phone,address,email):
        self.studentID = id
        self.lName = lName
        self.fName = fName
        self.status = status
        self.termStart = termStart
        self.termEnd = termEnd
        self.peopleSoftID = peopleSoft
        self.dob = dob
        self.phone = phone
        self.address = address
        self.email = email

File: src/app/_models/student.ts
import {User} from './user';
/**
 * Created by Cory Ginsberg on 5/4/2017.
 *
 * Student information
 *
 */

export class Student extends User {
    // TODO: remove 'id' and simply use DePaulID as the ID.
    id:              number;
    fullName:        string;
    depaulID:        number;
    status:          string; // Active, Graduated, or Frozen
    termStart:       Date;
    termEnd?:        Date;
    dateofBirth?:    Date;
    email:           string;
    phone?:          number;
    address?:        string;
    username:        string;
    password:        string;
}
